CASMREL-847/README.md

```diff
@@ -0,0 +1,10 @@
+# Disable JndiLookup.class in strimzi operator
+
+This hotfix applies to CSM 0.9.X (Shasta 1.4) and CSM 1.0.X (Shasta 1.5)
+
+This fix Updates the strimzi operator 0.15.0 container to 0.15.0-noJndiLookupClass.
+This container contains a version of log4j with the JdniLookup.class removed.
+
```

Comment:
I'd add a note here about security scans by tools like trivy and anchore/grype still flagging the image with log4j-related vulnerabilities. Feel free to include details from the scan results I posted in [CASMREL-847](https://jira-pro.its.hpecorp.net:8443/browse/CASMREL-847).